test/common/helpers/login-helpers.unit.spec.js

```diff
@@ -0,0 +1,47 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import { handleVerify } from '../../../src/js/common/helpers/login-helpers.js';
+
+// Does this need to be a React class in order for something to magically call this.serverRequest()?
```

Comment:
Ah, right...it's just a handle. There seems to be a lot of buzz about how to cancel/abort a promise chain which I will not attempt to understand right now. Would be glad to just get some functioning unit tests on this.